src/status_im/ui/screens/status/views.cljs

```diff
@@ -107,15 +107,17 @@
          [photos/member-photo from])]]
      [react/view {:flex 1}
       [react/view {:flex-direction  :row
-                   :justify-content :space-between}
+                   :justify-content :space-between
+                   :width           :98%
```

Comment:
could you please elaborate on this change ? does that mean all items will be 2% shorter ?

Reply:
what of there will be case where we need 4% ?

Reply:
Hi @flexsurfer : In some android layouts as seen in https://github.com/status-im/status-react/issues/11543 The timestamps would go out of the visible screen, hence I decided to restrict the width of the parent view to 98%

Reply:
could you elaborate please why 98% ?

Reply:
hi :) and thank you for the contribution ;)

Reply:
Sure, the logic behind 98% of available width is that we leave out 2% space as a margin on the right hand side and we only occupy 98% of available space to display the username and timestamp, this ensures that an overflow does not result in data going out of the screen( as observed in https://github.com/status-im/status-react/issues/11543 ) 

If you look at the screenshot I have share in the comments below it would show how the UI now looks like.

Reply:
but that means that now timestamp is shifted and has a bigger margin, and I'm still not sure if all cases are covered or there still will be the case when data goes out of the screen, what i mean is that this fix is for this specific case, and is not a general solution, and also timestamp now is not on its place its shifted on 2% to the left

Reply:
@flexsurfer : Thanks for taking the time to let me know your thoughts on this!

setting a fixed width to the parent view combined with a word-break property should ensure that textual content within that view stays within the defined limits.
Reasoning behind setting a 98% width was to ensure that most android devices have different viewport sizes and we could account for that. 

I have highlighted the entire width of the React View element in discussion here to show you how much space it takes up on the screen.

![IMG_8982](https://user-images.githubusercontent.com/64726664/158349915-2168ac32-7254-4c6a-957b-4d9e54533804.jpg)

This bug  [also ](https://github.com/status-im/status-react/issues/11543) appears only in the specific case when someone has set a long username and when a user sets a max length nickname for someone in their contacts.
After re-creating the issue I tested my fix on it and only then I have raised a PR for this issue.

If you have any different approach in mind or any scenario in which this would fail please let me know of that use case and I will plan my solution around it.

Reply:
thank you for the detailed explanation